hackathon/models.py

```diff
@@ -80,6 +80,12 @@ class Hackathon(models.Model):
     )
     is_public = models.BooleanField(default=True)
     max_participants = models.IntegerField(default=None, null=True, blank=True)
+    is_register = models.BooleanField(default=True)
+    google_registration_form = models.URLField(
```

Comment:
It's fine as it is for now, but for the future when adding fields I'd be more general and call it soemthing like "registration_url" or "registration_form" since it might not always be a Google Form. No need to change it right now, we can always change it in the future if need be.

Reply:
updated to registration_form in model, form, template, view and tested.